go.mod

```diff
@@ -0,0 +1,64 @@
+module github.com/docker/distribution
```

Comment:
We can't use a go.mod for v2, as that requires the module to be renamed to `github.com/docker/distribution/v2`

Reply:
Yes, my bad. I'm working on my repo to fix this.